CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 1.5.0 (released to-be-determined)
```

Comment:
moderate: I'd say is a patch version. We are not introducing any backwards incompatible change nor new feature. WDYT?

Reply:
Well for someone who is using Python 2 the change is sort of incompatible, thats why I didn't make it a patch.